packages/cml/src/replaceCmlPlaceholders/replaceCmlPlaceholders.ts

```diff
@@ -48,6 +48,7 @@ const replacePropValues = (mdx: string, values: Record<string, string> = {}): st
       case 'Image':
       case 'Images':
       case 'MediaBlock':
+      case 'Tag':
```

Comment:
please add a unit test that verifies this behaviour

Reply:
Done!